packages/page-claims/src/Warning.tsx

```diff
@@ -22,21 +22,21 @@ export interface Props{
 
 function Warning ({ className }: Props): React.ReactElement<Props> | null {
   const { allAccounts } = useAccounts();
-  const { api, isApiReady } = useApi();
+  const { api } = useApi();
   const { t } = useTranslation();
 
   // Find accounts that need attest. They are accounts that
   // - already preclaimed,
   // - didn't sign the attest yet.
   // `claims.preclaims` returns Some() for these accounts.
   const needsAttestArray = useCall<string[]>(api.query.claims.preclaims?.multi, [allAccounts], {
-     transform: (preclaims: Option<EthereumAddress>[]) =>
-       preclaims
-         .map((opt, index) => [opt.isSome, allAccounts[index]])
-         .filter(([isSome]) => isSome)
-         .map(([, address)) => address) 
+    transform: (preclaims: Option<EthereumAddress>[]) =>
+      preclaims
+        .map((opt, index) => [opt.isSome, allAccounts[index]])
+        .filter(([isSome]) => isSome)
+        .map(([, address]) => address as string)
```

Comment:
This also works (the first one defining the types) -

```suggestion
        .map((opt, index): [boolean, string] => [opt.isSome, allAccounts[index]])
        .filter(([isSome]) => isSome)
        .map(([, address]) => address)
```

I sometimes forget that TS is pretty good and then "not-so-good".

Reply:
ah nice thanks, this is much nicer